integreat_cms/cms/views/feedback/region_feedback_actions.py

```diff
@@ -199,14 +199,10 @@ def export_region_feedback(
     )
 
     resource = FeedbackResource()
-    dataset = Dataset()
-
-    headers = resource.export().headers
-    dataset.headers = headers
-
-    for obj in selected_feedback:
-        data = resource.export_resource(obj)
-        dataset.append(data)
+    dataset = Dataset(
+        (resource.export_resource(obj) for obj in selected_feedback),
```

Comment:
I think these should be unpacked as args
```suggestion
        *(resource.export_resource(obj) for obj in selected_feedback),
```